src/Integration/WPF/RelayCommand.cs

```diff
@@ -49,12 +50,25 @@ public RelayCommand(Action execute, Func<bool> canExecute)
 
         public bool CanExecute()
         {
-            return this.canExecute?.Invoke() ?? true;
+            try
+            {
+                return this.canExecute?.Invoke() ?? true;
+            }
+            catch (Exception ex) when (!ErrorHandler.IsCriticalException(ex))
+            {
+                return false;
+            }
         }
 
         public void Execute()
         {
-            this.execute();
+            try
+            {
+                this.execute();
+            }
+            catch (Exception ex) when (!ErrorHandler.IsCriticalException(ex))
+            {
+            }
```

Comment:
Could add a comment here and the other empty `catch` to remove the code smells
e.g.
`// Just squash the exception`